src/EncounterPage/EncounterPage.js

```diff
@@ -5,9 +5,9 @@ import PlayerEntry from './PlayerEntry'
 export default function EncounterPage() {
   return (
     <>
-      <EnemyEntry enemy={'Enemy 1'} maxHP={50} ac={17} />
+      <EnemyEntry enemy={'Enemy 1'} maxHP={50} HP={50} ac={17} />
```

Comment:
If the prop value is a string like "Enemy 1" , you don't need to wrap it in curly braces. Just use "".